website/static/js/koHelpers.js

```diff
@@ -207,13 +207,15 @@ ko.bindingHandlers.fadeVisible = {
     }
 };
 
+var tooltip = function(el, valueAccessor) {
+    var params = valueAccessor();
+    $(el).tooltip(params);
+};
 // Run Bootstrap tooltip JS automagically
 // http://getbootstrap.com/javascript/#tooltips
 ko.bindingHandlers.tooltip = {
-    init: function(el, valueAccessor) {
-        var params = valueAccessor();
-        $(el).tooltip(params);
-    }
+    init: tooltip,
+    update: tooltip
```

Comment:
Can you clarify the expected use case for an update callback being needed on a tooltip?

There's no *requirement* to provide both callbacks- one or the other is allowed, per http://knockoutjs.com/documentation/custom-bindings.html